database/Makefile

```diff
@@ -6,10 +6,11 @@ DB_USER ?= user_service_admin
 DB_PASSWORD ?= "comes-this-from-environment"
 MIGRATION_STEPS ?= 1
 
-URL_ENCODED_PASSWORD=$(shell urlencode ${DB_PASSWORD})
```

Comment:
Not strictly necessary for this PR but seems needed in case the password contains some symbols that can be interpreted by the shell.